trust-provider/trust-provider.md

```diff
@@ -0,0 +1,89 @@
+# Trust Provider
+Trust dApp browser provides simple a API for **dApp** developers to create multi-chain applications. Currently, our API allows you to get accounts and sign transactions [for any blockchain](https://github.com/TrustWallet/wallet-core/blob/master/docs/coins.md) for both iOS and Android.
+
+__Supported Coins__
+
+<a href="https://binance.com" target="_blank"><img src="https://raw.githubusercontent.com/TrustWallet/tokens/master/coins/714.png" width="32" /></a>
+<a href="https://ethereum.org" target="_blank"><img src="https://raw.githubusercontent.com/TrustWallet/tokens/master/coins/60.png" width="32" /></a>
```

Comment:
What about Callisto, GoChain, etc?

Reply:
Do we support them?

Reply:
it's an ethereum based chain, should be supported as well